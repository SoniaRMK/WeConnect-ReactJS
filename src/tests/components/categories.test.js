import {shallow} from 'enzyme';
import React from 'react';
import Categories from '../../components/business/categories';

describe('Categories component',() => {

    it('should match snapshot',() =>{
        const wrapper = shallow(<Categories/>)
        expect(wrapper).toMatchSnapshot();
    })
})
