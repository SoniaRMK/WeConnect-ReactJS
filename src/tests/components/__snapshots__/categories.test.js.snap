// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Categories component should match snapshot 1`] = `
<select
  className="form-control mb-2 mr-sm-2"
  name="category"
>
  <option
    value=""
  >
    Business Category
  </option>
  <option
    value="Consulting"
  >
    Consulting
  </option>
  <option
    value="Telecommunications"
  >
    Telecommunications
  </option>
  <option
    value="Food and Beverages"
  >
    Food and Beverages
  </option>
  <option
    value="Computing and Technology"
  >
    Computing and Technology
  </option>
  <option
    value="Hotels and Accommodation"
  >
    Hotels and Accommodation
  </option>
  <option
    value="Arts and Crafts"
  >
    Arts and Crafts
  </option>
</select>
`;
